assets/js/components/DashboardMainApp.js

```diff
@@ -249,6 +249,9 @@ export default function DashboardMainApp() {
 				<HelpMenu />
 			</Header>
 			<ConsentModeSetupCTAWidget />
+
+			<OverlayNotificationsRenderer />
+
```

Comment:
Same as above. It looks like rendering this component has already been addressed in `develop`. We should merge `develop` into this branch and accept incoming changes.